### PR TITLE
 Added Rakefile stub

### PR DESCRIPTION
**Pull request by jasnow**
Added Rakefile stub

## Motivation

To be more approachable to new Ruby developers.

### Test plan

When you run "rake", you get:
```
Run 'bazel test //... --config=dbg' to run the tests. More details in README file.
```

### CODE REVIEW

File: main/options/options.cc [hunk 1]
```diff
@@ -276,7 +276,7 @@ buildOptions(const vector<pipeline::semantic_extension::SemanticExtensionProvide
     // Used to populate default options.
     Options empty;
 
-    cxxopts::Options options("sorbet", "Typechecker for Ruby");
+    cxxopts::Options options("srb", "Typechecker for Ruby");
```
Comment (DarkDimius) [inline main/options/options.cc]: Could you please remove this from the PR?
for the same reason we've pushed back against it in past: it'll work incorrecly in `srb` too, as you want `srb tc`(or other wrapper commands).

The proper way it to allow to override command name via options

Comment (jasnow) [reply main/options/options.cc]: sorry

File: test/cli/help/help.out [hunk 2]
```diff
@@ -4,7 +4,7 @@ You must pass either `-e` or at least one folder or ruby file.
 
 Typechecker for Ruby
 Usage:
-  sorbet [OPTION...] <path 1> <path 2> ...
+  srb [OPTION...] <path 1> <path 2> ...
```
Comment (DarkDimius) [inline test/cli/help/help.out]: this needs to be reverted too to pass tests

**Comment by DarkDimius** [comment]
Thank you!

**Comment by jasnow** [comment]
You are welcome.

File: Gemfile [hunk 3]
```diff
@@ -0,0 +1,2 @@
+gem 'sorbet', :group => :development
```
Comment (DarkDimius) [inline Gemfile]: What is the gemfile for?

Comment (jasnow) [reply Gemfile]: I will remove it

**Comment by jasnow** [comment]
Anything else I need to do.

**Comment by DarkDimius** [comment]
Thank you!

**Comment by jasnow** [comment]
Thanks

**Pull request merged**